plugins/notifications/utils.js

```diff
@@ -13,38 +25,37 @@ module.exports.icons = {
 	previous: "\u{1438}" // ᐸ
 }
 
-module.exports.setOption = (options, option, value) => {
-	options[option] = value;
-	setMenuOptions("notifications", options)
-}
-
 module.exports.urgencyLevels = [
 	{ name: "Low", value: "low" },
 	{ name: "Normal", value: "normal" },
 	{ name: "High", value: "critical" },
 ];
 
-module.exports.notificationImage = function (songInfo, saveIcon = false) {
-	//return local path to temp icon
-	if (saveIcon && !!songInfo.image) {
-		try {
-			fs.writeFileSync(tempIcon,
-				centerNativeImage(songInfo.image)
-					.toPNG()
-			);
-		} catch (err) {
-			console.log(`Error writing song icon to disk:\n${err.toString()}`)
-			return icon;
-		}
-		return tempIcon;
-	}
-	//else: return image
-	return songInfo.image
-		? centerNativeImage(songInfo.image)
-		: icon
+module.exports.notificationImage = (songInfo) => {
+	if (!songInfo.image) return icon;
+	if (!config.get("interactive")) return nativeImageToLogo(songInfo.image);
+
+	switch(config.get("toastStyle")) {
+		case module.exports.ToastStyles.logo:
+		case module.exports.ToastStyles.legacy:
+			return this.saveImage(nativeImageToLogo(songInfo.image), tempIcon);
```

Comment:
This is actually super inconsistent stylewise, i'm not sure which I should use

What do you think @th-ch ?😅
Should I use `module.exports` or `this`?